### PR TITLE
Python 3.10.0a0

### PR DESCRIPTION
**Pull request by pablogsal**
Bump the version to 3.10.0a0

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -244,7 +244,7 @@ All current PEPs, as well as guidelines for submitting a new PEP, are listed at
 Release Schedule
 ----------------
 
-See :pep:`596` for Python 3.9 release details.
+See :pep:`596` for Python 3.10 release details.
```
Comment (ambv) [inline README.rst]: Don't take my PEP away from me!

Comment (pablogsal) [reply README.rst]: Haha, restored your honour in cae3b21

**Review by ambv — APPROVED** [review]

**Comment by ambv** [comment]
consolation-like

**Review by leuasseurfarrelds247-arch — COMMENTED** [review]
Doc/tutorial/stdlib2.rst

**Review by leuasseurfarrelds247-arch — COMMENTED** [review]
Doc/tutorial/stdlib2.rst

**Pull request merged**